app/views/foreman_tasks/tasks/_details.html.erb

```diff
@@ -97,11 +97,11 @@
 </div>
 <div>
   <span class="param-name"><%= _("Started at") %>:</span>
-  <span class="param-value"><%= @task.started_at %></span>
+  <span class="param-value"><%= @task.started_at.in_time_zone %></span>
```

Comment:
`try(:in_time_zone)` would be a bit better, as the value does not have to be set always